user-guide/Standard_Apps/TAG/Installing_TAG_App.md

```diff
@@ -0,0 +1,81 @@
+---
+uid: Installing_TAG_App
+---
+
+# Installing the DataMiner TAG Management app
+
+## Prerequisites
+
+- DataMiner version 10.4.2 or higher

```

Comment:
@sebastianulloa, what is the minimum DataMiner version required to use the TAG Management application?

Reply:
Minimum version has been updated. It is 10.4.5